axis.py

```diff
@@ -203,7 +183,9 @@ def _make_request(self, url, method, headers, data=None, params=None,
 
 class MockAxis360API(Axis360API):
 
-    def __init__(self, _db, with_token=True, *args, **kwargs):
+    @classmethod
+    def mock_collection(self, _db):
+        """Create a mock Axis 360 collection for use in tests."""
```

Comment:
I changed all the mock APIs to split out the code that creates a mock collection, so that it can be called separately.